lib/libvfio-user.c

```diff
@@ -551,6 +550,10 @@ add_fd_index(int *out_fds, size_t *nr_out_fds, int fd_search)
     assert(out_fds != NULL);
     assert(nr_out_fds != NULL);
 
+    if (fd_search < 0) {
```

Comment:
add behaviour to comment above? Also check for -1 explicitly maybe? And assert for any other negative value?

Reply:
ok